src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.Scanner;
+import java.util.ArrayList; 
+import java.text.ParseException;
+
 public class Duke {
-    public static void main(String[] args) {
+    void logo(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+    static void greeting(){
+        String output = "    ____________________________________________________________\n" +
+        "     Hello! I'm Duke\n" + 
+        "     What can I do for you?\n" +
+        "    ____________________________________________________________\n";
+        System.out.println(output);
+    }
+    static String addDoubleLine(String str){
+        String line = "    ____________________________________________________________";
+        return line + "\n" + str + "\n" + line;
+    }
+    static void handleList(ArrayList<Task> ls){
+        String output = ""; 
+        for(int i = 0; i < ls.size(); i++){
+            output = output + "     " +  (i+1) + ": " + ls.get(i).toString();
+            if(i < ls.size() - 1){
+                output = output + "\n";
+            }
+        }
+        System.out.println(addDoubleLine(output));
+    }
+    static void printMsg(Task t, int size){
+        System.out.println(addDoubleLine("     Got it. I've added this task: \n" + "      " + t.toString() + "\n     Now you have " + size + " tasks in the list."));
+    }
+    static void handleInput(){
+        Storage storage = new Storage();
+        ArrayList <Task> list = storage.recoverTasks();
+        Scanner sc = new Scanner(System.in);
+        while (sc.hasNextLine()){
+            try{
+                String line = sc.nextLine().toLowerCase();
+                String[] splited = line.split(" ");
+                String check = splited[0];
+                //bye
+                if (check.equals("bye")) {
```

Comment:
possible to change this to a switch statement. I did it this way too. But would have made it a switch statement on hind sight